### PR TITLE
Add share buttons to saved scenario page

### PR DESCRIPTION
**Pull request by noracato**
Hi!
We're in the process of adding functionality and design to the saved scenario page. This PR is a little more on the functionality side, because there is no design to fit this one in yet :)

### CODE REVIEW

File: app/views/saved_scenarios/show.html.haml [hunk 1]
```diff
@@ -28,7 +28,47 @@
...
+    = link_to "mailto: |
+              ?subject=#{t('scenario.share.email_subject')} #{@scenario&.title} |
+              &body=#{t('scenario.share.message')} #{request.original_url}" do |
+      .icon{ class: 'fa fa-envelope-o'}
+      Email
+    = link_to "http://www.twitter.com/share?url=#{request.original_url}",
+              target: "_blank" do
+      .icon{ class: 'fa fa-twitter'}
+      Twitter
+    = link_to "http://www.linkedin.com/shareArticle?mini=true |
```
[the remark this answers is not in the document]
Comment (noracato) [reply app/views/saved_scenarios/show.html.haml]: Also in this case of urls?

[the remark this answers is not in the document]
Comment (antw) [reply app/views/saved_scenarios/show.html.haml]: Yeah: some of the share URLs look pretty complicated (especially the e-mail one). I'd be inclined to stick them in helper methods as they make the template quite complicated.

Also, be sure to add `rel="noopener noreferrer"` if using `target="_blank"`: https://www.jitbit.com/alexblog/256-targetblank---the-most-underestimated-vulnerability-ever/

[the remark this answers is not in the document]
Comment (noracato) [reply app/views/saved_scenarios/show.html.haml]: Thanks! I'll move them to helper methods 👍

File: app/views/saved_scenarios/show.html.haml [hunk 2]
```diff
@@ -28,7 +28,41 @@
     %li
       %strong #{t('scenario.end_year')}:
       = @scenario.end_year
+  .sharing_links
+    = t('scenario.share.by')
+    .copy_link
+      %a
+        .icon{ class: 'fa fa-copy' }
+        Copy link
```
Comment (sfp-kemp) [inline app/views/saved_scenarios/show.html.haml]: I think this should be localized.

File: app/views/saved_scenarios/show.html.haml [hunk 3]
```diff
@@ -28,7 +28,41 @@
...
+      LinkedIn
+    = link_to share_by_facebook_link, open_attributes do
+      .icon{ class: 'fa fa-facebook-official' }
+      Facebook
 
   %p
     %small{:style => "font-size: 12px; font-weight: normal"}
       = t("scenario.overwritten")
+
+:javascript
```
Comment (sfp-kemp) [inline app/views/saved_scenarios/show.html.haml]: Not a huge fan of javascript in HTML templates. might be best to move this to its own file.

**Pull request merged**